### PR TITLE
Update Get-AppxPackage.md

### PR DESCRIPTION
**Pull request by RAJU2529**
Added new commands 
Get-AppxPackage -Name Microsoft.ScreenSketch

Get-AppxPackage -Publisher "CN=Microsoft Corporation, O=Microsoft Corporation, L=Redmond, S=Washington, C=US"

Get-AppxPackage -PackageTypeFilter Bundle
Get-AppxPackage -PackageTypeFilter Framework
Get-AppxPackage -PackageTypeFilter Main 
Get-AppxPackage -PackageTypeFilter Resource

The above commands are worked in windows 11 22h2 build no 22621.2361P

### CODE REVIEW

**Comment by RAJU2529** [comment]
@JohanFreelancer9 , Please review the sentences i added newly with examples added.
Thanking you

File: docset/winserver2022-ps/appx/Get-AppxPackage.md [hunk 1]
```diff
@@ -47,6 +47,34 @@ Get-AppxPackage -Name "Package17" -User "Contoso\EvanNarvaez"
...
+This command gives information about the ScreenSketch app.
+
+
+### Example 4: Get all app packages for a particular Publisher
+
+```powershell
+Get-AppxPackage -Publisher "CN=Microsoft Corporation, O=Microsoft Corporation, L=Redmond, S=Washington, C=US"
+```
+
+This command lists all the app packages that are installed from Microsoft Corporation on the computer.
```
Comment (JohanFreelancer9) [inline docset/winserver2022-ps/appx/Get-AppxPackage.md:65]:
```suggestion
This command lists all the Microsoft app packages that are installed on the computer.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

File: docset/winserver2022-ps/appx/Get-AppxPackage.md [hunk 2]
```diff
@@ -47,6 +47,33 @@ Get-AppxPackage -Name "Package17" -User "Contoso\EvanNarvaez"
 This command displays information about `Package17` if it's installed in the specified user
 profile.
 
+### Example 3: Get a particular app package information
+
+```powershell
+Get-AppxPackage -Name Microsoft.ScreenSketch
+```
+
+This command gives information about the ScreenSketch app.
```
Comment (get-itips) [inline docset/winserver2022-ps/appx/Get-AppxPackage.md:56]:
```suggestion
This command displays information about the ScreenSketch app.
```

File: docset/winserver2022-ps/appx/Get-AppxPackage.md [hunk 3]
```diff
@@ -47,6 +47,33 @@ Get-AppxPackage -Name "Package17" -User "Contoso\EvanNarvaez"
...
+
+### Example 4: Get all app packages for a particular Publisher
+
+```powershell
+Get-AppxPackage -Publisher "CN=Microsoft Corporation, O=Microsoft Corporation, L=Redmond, S=Washington, C=US"
+```
+
+This command lists all the Microsoft app packages that are installed on the computer.
+
+### Example 5: Get all app packages with PackageTypeFilter
```
Comment (get-itips) [inline docset/winserver2022-ps/appx/Get-AppxPackage.md:66]: According to the description of the parameter, we can specify with comma separated values like this, I tried and it works, so I think it is better to use that format

```
Get-AppxPackage -PackageTypeFilter bundle,framework,main,resource
```

Comment (get-itips) [reply docset/winserver2022-ps/appx/Get-AppxPackage.md:66]: I think we should use the comma separated value example or is it any special reason to run the cmdlet 4 times? Thanks

Comment (RAJU2529) [reply docset/winserver2022-ps/appx/Get-AppxPackage.md:66]: No special reason. They are all separated only. But users can choose any of those.

Comment (tiburd) [reply docset/winserver2022-ps/appx/Get-AppxPackage.md:66]: @RAJU2529 
Hello - Andres/get-itips (and I) work with Microsoft. 
Please incorporate his suggestions, and then we can merge this PR.
Thank you!

Comment (RAJU2529) [reply docset/winserver2022-ps/appx/Get-AppxPackage.md:66]: @tiburd  How do I reedit this article at present.

Comment (RAJU2529) [reply docset/winserver2022-ps/appx/Get-AppxPackage.md:66]: @JohanFreelancer9 , please review send suggestions edit , so that I ca accept it

Comment (tiburd) [reply docset/winserver2022-ps/appx/Get-AppxPackage.md:66]: Select the **Add a Suggestion** icon on the upper left of the comment box:
image

Comment (tiburd) [reply docset/winserver2022-ps/appx/Get-AppxPackage.md:66]: @JohanFreelancer9 
Please disregard the above comment. We'll take care of it.
Thanks!

Comment (tiburd) [reply docset/winserver2022-ps/appx/Get-AppxPackage.md:66]: I think like this:
image

And then for each of the following lines:
image

Comment (tiburd) [reply docset/winserver2022-ps/appx/Get-AppxPackage.md:66]: @RAJU2529 
If you have trouble, I will make the change. If you want me to do that, please provide the correct syntax for the comma-separated values.
Thank you!

Comment (RAJU2529) [reply docset/winserver2022-ps/appx/Get-AppxPackage.md:66]: @tiburd , you mean starting word should be in Capital ?

Comment (tiburd) [reply docset/winserver2022-ps/appx/Get-AppxPackage.md:66]: @RAJU2529 
Disregard. Thank you. I'll merge the PR after it passes the build checks.

Comment (RAJU2529) [reply docset/winserver2022-ps/appx/Get-AppxPackage.md:66]: @tiburd . Thanking you.

**Pull request merged**